configuration/pih/scripts/global/zl.js

```diff
@@ -88,3 +88,62 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
 
 }
 
+function datepickerStartAndEndDateValidation(badDateInTheFutureMsg,badStartDateGreaterThanEndDateMsg){
+
+  $("#startDate-0,#startDate-1").each(function(j, domEl){
+    $(this).change(function (e){
+      let startDate = Date.parse(e.target.value);
+      let endDate = Date.parse(jq(`#endDate-${j} input[type=text]`).val());
+
+        if (startDate > Date.now()){
+          jq(`#${domEl.getAttribute('id')} span`).show();
+          jq(`#${domEl.getAttribute('id')} span`).text(badDateInTheFutureMsg);
+          buttonsDisabled(true)
+        }else {
+          jq(`#${domEl.getAttribute('id')} span`).hide();
+          jq(`#${domEl.getAttribute('id')} span`).text('');
+          buttonsDisabled(false)
+        }
```

Comment:
Future dates are, by default, not allowed. I think you can delete this code.

Reply:
@brandones  for context, I think Debbie was asking on the ticket if this validation could happen prior to form submission, which is why I think Louidor asdded it here.

Reply:
Oh! Okay, all good then.